conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -30,21 +30,29 @@ pkgs_dirs:
   - /opt/conda/pkgs
 
 CONDARC
-{% if build_with_mambabuild %}
+{% if conda_build_tool == "mambabuild" %}
 {%- set CONDA_INSTALL_CMD="mamba" %}
-{%- set BOA="boa " %}
-{%- set BUILD_CMD="mambabuild" %}
+{%- set conda_build_tool_deps="boa " %}
+{%- set BUILD_CMD="conda mambabuild" %}
+{% elif conda_build_tool == "conda-build+conda-libmamba-solver" %}
+{%- set CONDA_INSTALL_CMD="mamba" %}
```

Comment:
We could use "conda" here right or does the solver switch only effect the build command and not the other ci setup commands?